metals/src/main/scala/scala/meta/internal/metals/Messages.scala

```diff
@@ -522,14 +523,16 @@ object Messages {
         MessageType.Info,
         s"Created .scalafmt.conf${tryAgain(isAgain)}."
       )
+    private val messageStart = "Unable to format since"
     def isCreateScalafmtConf(params: ShowMessageRequestParams): Boolean =
-      params.getMessage == createScalafmtConfMessage
-    def createScalafmtConfMessage: String =
-      s"Unable to format since this workspace has no .scalafmt.conf file. " +
-        s"To fix this problem, create an empty .scalafmt.conf and try again."
-    def params(path: AbsolutePath): ShowMessageRequestParams = {
+      params.getMessage.startsWith(messageStart)
+    def message(path: String): String = {
```

Comment:
I think we can leave this message. What was the reason for a change?

Reply:
It was changed to show the correct path to the config file that Metals tried to use.

A user might set a non-standard path for it in configuration and forget about that.
Then it will receive this notification and if we will decide to create `.scalafmt.conf` by hand nothing will be changed

Reply:
I would rather change the message depending on whether we have a default scalafmt or a custom path. I would keep the current message for the current situation and change it for the latter to just `Custom scalafmt configuration at $path does not exist`.

Reply:
Reverted this change